src/Command/ProcessCommand.php

```diff
@@ -101,6 +114,34 @@ protected function configure(): void
     protected function initialize(InputInterface $input, OutputInterface $output): void
     {
         $this->io = new SymfonyStyle($input, $output);
+
+        $limitPerResource = $input->getOption(self::OPTION_LIMIT);
+        Assert::nullOrIntegerish($limitPerResource);
+        if ($limitPerResource !== null) {
+            $limit = (int) $limitPerResource;
+            Assert::greaterThanEq($limit, 0, 'The limit can\'t be negative');
```

Comment:
I suppose this isn't completely correct if you don't do the other change I suggested with the `$this->limitPerResource` being not nullable. If not, then this should be `Assert::greaterThan`, shouldn't it?

Reply:
If I add limit=0 as input I except the limit to be 0. Not infinite.

Reply:
Please, can't we use `Assert::greaterThan`. A limit of `0` makes no sense

Reply:
You can also set a limit of 0 on a SQL query. Should MySQL complain that what you're doing makes no sense? Or all the paging APIs in the world?

I really, *really* do not get your opposition to a limit of 0.

So - I'll leave it in for now. Its up to you to sneak the change in another time 😁